packages/view3d/test/suite/core/PostProcessing.ts

```diff
@@ -0,0 +1,68 @@
+import { createView3D } from "../../test-utils";
+
+describe("PostProcessing", () => {
+  describe("bloom effect", () => {
+    it("should resolution be greater than or equal to canvas size", async () => {
+      const view3d = await createView3D({bloom: true});
+
+      view3d.on('load', () => {
+        const resolutionSize = view3d.postProcessing.bloomComponent.resolution;
+        const canvasSize = view3d.renderer.canvasSize;
+
+        expect(resolutionSize.width >= canvasSize.width).to.be.true;
+        expect(resolutionSize.height >= canvasSize.height).to.be.true;
+      });
+    });
+
+    it("should first pass of the composite is renderPass.", async () => {
```

Comment:
Please set the test names as full English sentences.
Like, `it("should set the render pass as the first pass of the composer")`

Reply:
Check it!